homeassistant/components/youless/sensor.py

```diff
@@ -69,8 +69,8 @@ class YouLessSensorEntityDescription(SensorEntityDescription):
         key="usage",
         device_group="power",
         device_group_name="Power usage",
-        name="Power Usage",
-        icon="mdi:meter-electric",
+        translation_key="active_power_w",
+        has_entity_name=True,
```

Comment:
Instead of adding `has_entity_name = True` to every description, add `_attr_has_entity_name = True` to the sensor class (outside of the constructor)